[Integration][GCP] | Optionally convert resource fields to camelCase for non cloud asset apis (#1118)
port-labs/ocean: integrations/gcp/CHANGELOG.md

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

<!-- towncrier release notes start -->

> ## 0.1.66 (2024-11-05)

Reviewer: ```suggestion

## 0.1.66 (2024-11-05)
```

Reviewer: Leave a space after `<!-- towncrier release notes start -->`

### Improvements

> - Sets GCP returned `pubsub` fields to `camelCase` rather than `snake_case`.  

Reviewer: Lets change to more precise changelog message since this doesn't specifically affect pub sub alone.

> 

Reviewer: ```suggestion

```

Reviewer: Leave two spaces before the next log

<!-- towncrier release notes start -->

## 0.1.76 (2024-12-05)

> 

Reviewer: ```suggestion

```

### Improvements

> - Sets GCP returned `pubsub` fields to `camelCase` rather than `snake_case`.  

Reviewer: ```suggestion
- Added `preserveApiResponseCaseStyle` selector to optionally convert resource fields to and from `snake_case` and `camelCase` for non-cloud asset APIs.
```
